.golangci.yml

```diff
@@ -223,25 +223,21 @@ linters:
     # - govet
     # - errcheck
     # - staticcheck
-    # - unused
+    - unused
     # - gosimple
-    # - structcheck
-    # - varcheck
```

Comment:
Why are these deleted? Are not we planning to enable them later?

Reply:
Please see the answer to the `deadcode` removal.